examples/scenes/Cargo.toml

```
@@ -18,7 +18,7 @@ instant = { workspace = true }
 # Used for the `download` command
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 byte-unit = "4.0.19"
-dialoguer = "0.10.4"
+inquire = "0.7"
```

I'd like to also specify the patch version so that `cargo upgrade` can edit it when needed.

```suggestion
inquire = "0.7.0"
```